plugins/kosync.koplugin/main.lua

```diff
@@ -393,6 +430,11 @@ function KOSync:setCustomServer(server)
     self.settings.custom_server = server ~= "" and server or nil
 end
 
+function KOSync:setHostname(hostname)
```

Comment:
Just 2 lines, the method isn't called anywhere else, can de done right in the callback.